RELEASE-NOTES.md

```diff
@@ -39,6 +39,7 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 - Inlet: Added support for deeply nested containers of structs
 - Inlet: Added support for `void` and strings in Lua-defined functions
 - Inlet: Added `get<std::vector<T>>` for retrieving arrays without index information
+- Inlet: Added a new `Writer` for generating JSON schemas
```

Comment:
Consider expanding this line a bit (but not too much). E.g. what can these be used for?